src/coreclr/nativeaot/BuildIntegration/Microsoft.NETCore.Native.targets

```diff
@@ -207,6 +209,7 @@ The .NET Foundation licenses this file to you under the MIT license.
       <IlcArg Include="@(IlcReference->'-r:%(Identity)')" />
       <IlcArg Include="@(MibcFile->'--mibc:%(Identity)')" />
       <IlcArg Condition="$(IlcGenerateMetadataLog) == 'true'" Include="--metadatalog:$(NativeIntermediateOutputPath)%(ManagedBinary.Filename).metadata.csv" />
+      <IlcArg Condition="$(PlatformTarget) != ''" Include="--targetarch:$(PlatformTarget)" />
```

Comment:
If this is just for the multifile build of the coreclr test tree, maybe we could just pass a PlatformTarget here:

https://github.com/dotnet/runtimelab/blob/ee45ce1bbeb58d982956120723b41eb13455bf08/src/tests/run.proj#L430

Reply:
Thank you for the hint. We should add a check that `PlatformTarget` is set and then we can remove this `Condition`. Going to do it the next time.